### PR TITLE
added new company "clusus" to companies.json

### PR DESCRIPTION
**Pull request by bntshakya**
## What kind of change does this PR introduce? 

- [ ] Bug Fix
- [ ] Documentation change
- [x] Added new company
- [ ] Other(s)

### CODE REVIEW

File: data/companies.json [hunk 1]
```diff
@@ -176,6 +176,22 @@
         "youtube": "https://www.youtube.com/c/CloudfactoryHQ"
       }
     },
+    {
+        "name": "Clusus Information Solutions",
+        "establishedAt": 1535414400,
+        "location": "Jhamsikhel, Lalitpur",
+        "website": "https://www.clusus.com/",
+        "email": "Info@clusus.com",
+        "phone": "+977-01-5400593",
```
Comment (susanhsrestha) [inline data/companies.json:185]: The phone number formatting is incorrect.
```suggestion
        "phone": "+977-15400593",
```

File: data/companies.json [hunk 2]
```diff
@@ -176,6 +176,22 @@
         "youtube": "https://www.youtube.com/c/CloudfactoryHQ"
       }
     },
+    {
+        "name": "Clusus Information Solutions",
+        "establishedAt": 1535414400,
+        "location": "Jhamsikhel, Lalitpur",
+        "website": "https://www.clusus.com/",
+        "email": "Info@clusus.com",
```
Comment (susanhsrestha) [inline data/companies.json:184]: Maybe lowercase on email helps.
```suggestion
        "email": "info@clusus.com",
```

**Review by susanhsrestha — CHANGES_REQUESTED** [review]

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]
Please update the PR title as well to reflect your changes. The title should be clear and concise and should reflect on the changes you made via the PR.

**Comment by bntshakya** [comment]
@susanhsrestha @thenishantsapkota requested changes have been made

File: data/companies.json [hunk 3]
```diff
@@ -448,6 +464,22 @@
...
+          "phone": "+977-012345678",
+          "socials": {
+            "facebook": "https://facebook.com",
+            "instagram": "https://instagram.com",
+            "github": "https://github.com",
+            "linkedin": "https://linkedin.com",
+            "twitter": "https://twitter.com",
+            "youtube": "https://youtube.com"
+          }
+    },
```
Comment (thenishantsapkota) [inline data/companies.json:482]: Why is this here?

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

File: data/companies.json (hunk 3 above)
Comment (bntshakya) [reply data/companies.json:482]: oh i didnt notice them i'll update it

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]
Please properly format the code using prettier.

**Review by susanhsrestha — APPROVED** [review]

**Review by thenishantsapkota — APPROVED** [review]
LGTM!
Thank you for your contribution @bntshakya
